sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1302,3 +1304,52 @@ def __init__(self, error=None, is_fatal=False, description=None, position=None):
         self.is_fatal = is_fatal
         self.description = description
         self.position = position
+
+class BlobAudience(object):
+    """Audiences available for Blobs.
+
+    :keyword str value: The Azure Active Directory audience to use when forming authorization scopes.
```

Comment:
A couple of things, this looks more like the docstring for the `__init__`? Also, you have `value` defined as a param currently, not keyword. I think that's fine, just have to update the dcostring.